api/checks/summaries/completed.md

```diff
@@ -23,6 +23,6 @@ Other links that might be useful:
 {{- if .MasterDiffURL }}
 - [`{{ printf "%.7s" .HeadRun.FullRevisionHash }}` vs latest master]({{ .MasterDiffURL }})
 {{- end }}
-- [Latest results for `{{ printf "%.7s" .HeadRun.FullRevisionHash }}`]({{.HostURL}}?sha={{.HeadRun.Revision}})
+- [Latest results for `{{ printf "%.7s" .HeadRun.FullRevisionHash }}`]({{.HostURL}}?sha={{.HeadRun.Revision}}&label=pr_head)
```

Comment:
I don't think the `pr_head` label is necessary -- the `HeadRun.Revision` is on a PR branch so it has to be a PR head run, but it doesn't matter either way.

Reply:
It is; this is an unrelated drive-by. Note that we "fake" the SHA for both runs as the SHA with changes, so there's a race condition.